packages/react/src/components/UIShell/SideNav.js

```diff
@@ -78,9 +99,9 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
         {...accessibilityLabel}
         onFocus={event => handleToggle(event, true)}
         onBlur={event => handleToggle(event, false)}
-        onMouseEnter={() => handleToggle(true)}
-        onMouseLeave={() => handleToggle(false)}>
-        {children}
+        onMouseEnter={() => handleToggle(true, true)}
```

Comment:
```suggestion
        onMouseEnter={event => handleToggle(event, true)}
```

Reply:
@asudoh - before my changes, `handleToggle` was called here with just true as the only argument. I can change it to whatever the event is as you suggest, but if someone downstream passes the `onToggle` property to the `SideNav` and expects the bool value, would that not be a breaking change for them?

Reply:
Thank you for putting your thoughts on this and elaborating it @matthew-chirgwin! I now see your point (I missed it, tricked by argument name), and I now think the best is keeping `handleToggle()` call intact - Meaning, call with one arg.

Reply:
Hi @asudoh - the only issue with moving to one argument is the effect that would then have on how I changed `handleToggle` to handle this case (lines 48-50). That change could be pushed into its own function, but would that add any value?

Reply:
Just leaving a note, though this is merged already; I think the best approach is making `handleToggle()` obvious that the first arg is optional and and the second arg becomes the first arg when the caller doesn't specify the first arg.